code-challenges.js

```diff
@@ -39,6 +87,29 @@ var testString1 = "albatross"
 // Expected output: "aLbAtRoSs"
 var testString2 = "jabberwocky"
 // Expected output: "jAbBeRwOcKy"
+const capitolize = (string) => {
+// In the case we were given a string of multiple words we would first put each word in its own array
+    let splitStr = string.split(" ")
+  let newArr = []
+// Then create an array fro word
+    for (var i = 0; i < splitStr.length; i++) {
+    var letters = splitStr[i].split("")
+// Find the value of each positive index
+    for (var i = 0; i < letters.length; i++) {
+      if (i % 2 == 0) {
+// Capitolise the value
+        letters[i] = letters[i].toUpperCase()
+// Else lower case the values of odd indexes
+      } else {
+        letters[i] = letters[i].toLowerCase()
+      }
+    }
+    newArr.push(letters.join(""))
+  }
+  return newArr.join(" ")
+}
+console.log(capitolize(testString1))
+console.log(capitolize(testString2))
```

Comment:
Almost there! I know I mention this a lot, but I'd like you to pay close attention to your expected outputs. Your code currently outputs the letters capitalized in the wrong order. 

In a real world situation like this, your code would be rejected and you'd have to go back and refactor for this, thus causing the company money(time) and ultimately a bit of bad recognition on your end as a coder.  So let's get this kind of stuff knocked out where it's safe, here in the assessments! 😄